store/query/utxo.go

```diff
@@ -28,28 +42,74 @@ func NewUtxoQuerier(utxoStore store.UTXOStore) sdk.Querier {
 		switch path[0] {
 		case QueryBalance:
 			if len(path) != 2 {
-				return nil, ErrInvalidPath("expected balance/<address>")
+				return nil, sdk.ErrUnknownRequest("exprected balance/<address>")
```

Comment:
use local error

Reply:
FWIW, i think for an invalid query, sdk.ErrknownRequest should suffice